src/transformers/__init__.py

```diff
@@ -3827,6 +3842,16 @@
     else:
         from .models.bert import TFBertTokenizer
 
+    try:
+        if not is_keras_nlp_available():
+            raise OptionalDependencyNotAvailable()
+    except OptionalDependencyNotAvailable:
+        print("except")
```

Comment:
```suggestion
```

Reply:
Yeah sorry for that debugging goes brrr sometimes.

Reply:
Done.